ddl/db_test.go

```diff
@@ -4548,6 +4548,30 @@ func (s *testSerialDBSuite) TestDDLJobErrorCount(c *C) {
 	}
 }
 
+func (s *testDBSuite1) TestAlterTableWithValidation(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test")
+	s.tk.MustExec("drop table if exists t1")
+	defer s.tk.MustExec("drop table if exists t1")
+
+	s.tk.MustExec("create table t1 (c1 int, c2 int as (c1 + 1));")
+	s.tk.MustExec("alter table t1 with validation")
+	c.Assert(s.tk.Se.GetSessionVars().StmtCtx.WarningCount(), Equals, uint16(1))
+	s.tk.MustQuery("show warnings").Check(testutil.RowsWithSep("|", "Warning|8200|ALTER TABLE WITH VALIDATION is currently unsupported"))
+}
+
+func (s *testDBSuite2) TestAlterTableWithoutValidation(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test")
+	s.tk.MustExec("drop table if exists t1")
+	defer s.tk.MustExec("drop table if exists t1")
+
+	s.tk.MustExec("create table t1 (c1 int, c2 int as (c1 + 1));")
+	s.tk.MustExec("alter table t1 without validation")
```

Comment:
Could we put this test to `TestAlterTableWithValidation`? Then we can reduce the time to run the test.

Reply:
All right.